django/applications/catmaid/control/ontology.py

```diff
@@ -68,9 +68,10 @@ def __str__(self):
         return self.__unicode__()
 
     def __unicode__(self) -> str:
-        return "[CA {}: {} R {}: {} CB {}: SC {}: {}]".format(self.class_a.id,
-                self.class_a, self.relation.id, self.relation, self.class_b.id,
-                self.class_b, self.super_class.id, self.super_class)
+        return f"[CA {self.class_a.id}: {self.class_a} " + \
+               f"R {self.relation.id}: {self.relation} " + \
+               f"CB {self.class_b.id}: {self.class_b} " + \
```

Comment:
NOTE: This is worth some attention! The old code's has what looks to be a bug in its formatting that caused CB's values to leak into later reporting (it provides two CB stats in the arg list but the string only consumes one). Fixing it felt like the right thing.

Reply:
Agreed. This formatting looks much better too. Do we even need a `__unicode__` method in py3? It may be that we can just put that block straight into `__str__` rather than having the redirection.

Reply:
That's a good question; I don't know the answer.

Reply:
According to this, we may only need __str__() in py3:
https://docs.djangoproject.com/en/1.11/topics/python3/#str-and-unicode-methods

Reply:
:+1: